#+title: About Me
#+author: Gabriel Lozano
#+startup: logdrawer
#+startup: preview
#+OPTIONS: toc:nil num:nil
#+BEGIN_EXPORT html
---
layout: page
title: "About Me"
subtitle: "Gabriel Lozano"
permalink: /:title/
tags: [test, another test, databases]
---
#+END_EXPORT

I am a mathematician and computer scientist from Colombia that is currently making his graduate studies in Paris.

* Interests
I am mainly interested in databases.
* Work experience
I have worked in Data science projects for some years while I was doing my undergraduate degree. I was working closely with the COMBIOS lab.

I worked for almost two years at Instaleap as a Machine Learning Engineer.
* Education
** Bachelor
I did a double degree of Mathematics and Computer Science in the Universidad Nacional de Colombia from 2016 to 2021.
** Masters
I am currently doing an Erasmus+ for a masters degree in Big Data Management and Analytics.

* Publications
** Nonparametric clustering of RNA‐sequencing data
Lozano, G., Atallah, N., & Levine, M. (2023). Nonparametric clustering of RNA‐sequencing data. Statistical Analysis and Data Mining: The ASA Data Science Journal, 16(6), 547-559.

Identification of clusters of co-expressed genes in transcriptomic data is a difficult task. Most algorithms used for this purpose can be classified into two broad categories: distance-based or model-based approaches. Distance-based approaches typically utilize a distance function between pairs of data objects and group similar objects together into clusters. Model-based approaches are based on using the mixture-modeling framework. Compared to distance-based approaches, model-based approaches offer better interpretability because each cluster can be explicitly characterized in terms of the proposed model. However, these models present a particular difficulty in identifying a correct multivariate distribution that a mixture can be based upon. In this manuscript, we review some of the approaches used to select a distribution for the needed mixture model first. Then, we propose avoiding this problem altogether by using a nonparametric MSL (maximum smoothed likelihood) algorithm. This algorithm was proposed earlier in statistical literature but has not been, to the best of our knowledge, applied to transcriptomics data. The salient feature of this approach is that it avoids explicit specification of distributions of individual biological samples altogether, thus making the task of a practitioner easier. We performed both a simulation study and an application of the proposed algorithm to two different real datasets. When used on a real dataset, the algorithm produces a large number of biologically meaningful clusters and performs at least as well as several other mixture-based algorithms commonly used for RNA-seq data clustering. Our results also show that this algorithm is capable of uncovering clustering solutions that may go unnoticed by several other model-based clustering algorithms. Our code is publicly available on Github [[https://github.com/Matematikoi/non_parametric_clustering][HERE]].
** Characterization of skin patterns in Pseudoplatystoma Magdaleniatum
Lozano, G., Hernández, D., Chaves, N., Valderrama, M., Mojica, J., & Gómez, F. (2017, December). Characterization of skin patterns in Pseudoplatystoma Magdaleniatum. In 2017 Sustainable Internet and ICT for Sustainability (SustainIT) (pp. 1-3). IEEE.

Pseudoplatystoma Magdaleniatum is one of the most representative fish species from the Magdalena river basin in Colombia. This endemic catfish is highly valuable from the economic and environmental perspectives. Nevertheless, it is currently under threat because of the indiscriminate fishery and the degradation of their ecological niche. Strategies for conservation require a detailed individual description. Traditional characterization methods are invasive and time expensive. Here we introduce a novel automatic method to segment and describe the characteristic spot and stripe skin pattern of this specie. The method was evaluated on a photographic database of 50 individuals, which characteristic skin patterns were manually segmented by a trained expert. The proposed approach provides precise skin pattern segmentations and informative geometrical features of these patterns.
